pkg/samples/samples.go

```diff
@@ -190,15 +190,13 @@ func (s *SampleManager) Initialize(app string) error {
 	} else {
 		err := s.Git.Pull(appPath)
 		if err != nil {
-			if err != nil {
```

Comment:
I'm not sure why we had both of these `if` statements, they were redundant and go was complaining